src/app/features/group-list/group-item/group-item.component.ts

```diff
@@ -0,0 +1,21 @@
+import { ChangeDetectionStrategy, Component, Input } from '@angular/core';
+import { MatButtonModule } from '@angular/material/button';
+import { MatIconModule } from '@angular/material/icon';
+
+export interface GroupItem {
+  id: number;
+  name: string;
+  description: string;
+}
+
+@Component({
+  selector: 'app-group-item',
```

Comment:
jeśli mamy wymagany input to obowiązkowo trzeba zawężyć nasz selektor

Reply:
Zawężony selektor komponentu